docs/DataAlternative.puml

```diff
@@ -0,0 +1,11 @@
+@startuml
+
+hide attributes
+hide methods
+
+ModuleList --> "*" Module
```

Comment:
Nope, this diagram is to explain an "alternative" way of implementing "general tasks" as only a TaskList, and why it is worse (because we need to interact with a Module or TaskList depending on whether it's a general task or not)